<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartSettings</class>
 <widget class="QDialog" name="SmartSettings">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3840</width>
    <height>2160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smartsettings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="files.qrc">
    <normaloff>:/resources/icons/64x64/simplest-studio.png</normaloff>:/resources/icons/64x64/simplest-studio.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_main">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3840</width>
       <height>2160</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="rightMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBox_compression">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <property name="maxCount">
         <number>20</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 (Standart)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBox_preference">
        <property name="text">
         <string>Give preference to quality in doubtful cases</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_bit_depth">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxCount">
         <number>20</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>16 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 bit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_compression">
        <property name="text">
         <string>Compression:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Optimize for:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox_content_type">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3840</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>6</number>
        </property>
        <property name="maxCount">
         <number>20</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Telephone - 8 kHz - Lowest Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speech - 11,025 kHz - Lowest Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speech - 16 kHz - Low Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Radio - 22,05 kHz - Low Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Radio - 24 kHz - Low Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FM Radio - 32 kHz - Low Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CD Audio - 44.1 kHz - High Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Digital audio tape - 48 kHz - High Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVD Audio MLP 5.1 - 88.2 kHz - Studio Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVD Audio MLP 5.1 - 96 kHz - Studio Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVD Audio MLP 2.0 -176.4 kHz - Studio Quality</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVD Audio MLP 2.0 -192 kHz - Studio Quality</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_bit_depth">
        <property name="text">
         <string>Bit depth:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="buttonContinue">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Content type:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;   As a result of audio analysis, the following audio parameters are determined. To continue, click Continue or select the parameters manually.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox_optimize_for">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3840</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <property name="maxCount">
         <number>20</number>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="placeholderText">
         <string notr="true">0</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Listening in the car on high-quality equipment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Listening at home on standard equipment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Listening at home on high-quality equipment</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Studio quality</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="buttonCancel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
